C01/credit-batch-metadata/C01-20150101-20151231-003.json

```
@@ -0,0 +1,18 @@
+{
+  "@context": {
+    "schema": "http://schema.org/",
+    "regen": "http://regen.network/"

```

the `@context` is missing the following, ditto for all the other credit batch json
```suggestion
    "regen": "http://regen.network/",
    "regen:additionalCertifications": { "@container": "@list" }
```